source/sync/app-builder/tiered-device-sync/index.txt

```diff
@@ -0,0 +1,379 @@
+--------------------------------------------------------
+
+Set up and run the Tiered Sync server on the EC2 instance.
+
+.. include:: /includes/important-stop-tiered-sync-server.rst
+
+.. procedure::
+
+   .. step:: Get the Tiered Sync Server Code
+
+      Use ``wget`` to get the Tiered Sync Server code:
+
+      .. code-block:: shell
+
+         wget https://tiered-sync-package.s3.amazonaws.com/d49a2f331f53cc0e6af450a067c4fd9184c3ae63-tiered_sync.tgz
+
+      Unzip the tar to get the files.
+
+      .. code-block:: shell
+
+         tar -xvf *.tgz
+
+      You now have a ``tiered_sync`` directory containing the server files.
+
+   .. step:: Complete the Tiered Sync Configuration Details
+
+      The ``tiered_sync`` directory contains a :file:`config.json` file you 
+      must edit with the appropriate values to configure the server.
+
+      .. code-block:: json
+
+         {
+           "clientAppId": "YOUR-APP-ID",
+           "query": {
+             "YOUR-COLLECTION-NAME": "truepredicate"
+           },
+           "cloudSyncServerAuth": "YOUR-AUTH-TOKEN",
+           "hostname": "YOUR-TIERED-SYNC-SERVER-DOMAIN"
+         }
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Field name
+           - Value
+
+         * - ``clientAppId``
+           - The App ID of the App Services App you created above.
+
+          
+         * - ``YOUR-COLLECTION-NAME``
+           - The name of the collection in your database where the Tiered 
+             Sync server should sync data. If you're using a 
+             template app, this is the ``Item`` collection.
+             Currently, ``truepredicate`` is the only supported query.
+
+         * - ``cloudSyncServerAuth``
+           - This is the Tiered Sync Server authorization token you generated
+             and saved as the ``SECRET`` in a step above.
+
+
+         * - ``hostname``
+           - The public DNS of your EC2 instance. This should resemble something
+             like ``ec2-13-58-70-88.us-east-2.compute.amazonaws.com``.
+
+   .. step:: Install Dependencies
+
+      The Tiered Sync server requires several dependencies, which are listed 
+      in the :file:`README.md` in the ``tiered_sync`` directory. Follow the 
+      instructions to install the required dependencies.
+
+   .. step:: Start the Tiered Sync Server
+
+      To start the server, from the ``tiered_sync`` directory:
+
+      .. code-block:: shell
+
+         sudo make up
+
+      You may get a message similar to ``Couldn't connect to Docker daemon at http+docker://localunixsocket``.
+      If this occurs, you may need to add your user account to the docker group.
+
+      If you have successfully started the Tiered Sync server, you can see
+      the status of the server with:
+
+      .. code-block:: shell
+
+         sudo make status
+
+      Now you can connect clients directly to the Tiered Sync Server.
+
+      .. include:: /includes/important-stop-tiered-sync-server.rst
```

Comment:
I imagine you added this twice on purpose (once at the start of the section and here), but wanted to double check.

Reply:
I did, because I didn't think to do this and was surprised to find this out when I went back to do something else the following day. I was hoping that if I listed it at the beginning of the procedure and then again when you actually got the Tiered Sync server running, folks would see it in at least one of the places. I'm open to removing it in one of the places if it reads oddly being on the page twice, though.